### PR TITLE
Test: Improve Tesla LKAS Passthru Test Coverage

### PR DESCRIPTION
**Pull request by ssysm**
- Cover test that make sure CAN is not blocked during autopark
- Cover test for rising edge LKAS forwarding
- Cover test for stock LKAS is not allowed when OP are enabled

### CODE REVIEW

File: opendbc/safety/tests/test_tesla.py [hunk 1]
```diff
@@ -280,6 +282,22 @@ def test_stock_lkas_passthrough(self):
...
+    self.assertEqual(1, self._rx(no_lkas_msg_cam))
+    self.assertEqual(-1, self.safety.safety_fwd_hook(2, no_lkas_msg_cam.addr))
+    self.assertTrue(self._tx(no_lkas_msg))
+    self.safety.set_controls_allowed(True)
+    self.assertTrue(self._rx(lkas_msg_cam))
+    self.assertEqual(-1, self.safety.safety_fwd_hook(2, lkas_msg_cam.addr))
+    self.assertFalse(self._tx(lkas_msg_cam))
+
+
+
```
Comment (sshane) [inline opendbc/safety/tests/test_tesla.py]:
```suggestion
```
